Misc/NEWS.d/next/Build/2020-12-22-17-57-04.bpo-31904.j3a8r0.rst

```diff
@@ -0,0 +1 @@
+Add --enable-test-modules option for configure.
```

Comment:
Please document it also in https://docs.python.org/dev/whatsnew/3.10.html#build-changes (Doc/whatsnew/3.10.rst).

Reply:
Done.